bskyweb/cmd/bskyweb/server.go

```diff
@@ -292,6 +302,30 @@ func (srv *Server) Download(c echo.Context) error {
 	return c.Redirect(http.StatusFound, "/")
 }
 
+// Handler for proxying top-level paths to link service, which ends up serving a redirect
+func (srv *Server) LinkProxyMiddleware(url *url.URL) echo.MiddlewareFunc {
+	return middleware.ProxyWithConfig(
+		middleware.ProxyConfig{
+			Balancer: middleware.NewRoundRobinBalancer(
+				[]*middleware.ProxyTarget{{URL: url}},
+			),
+			Skipper: func(c echo.Context) bool {
+				req := c.Request()
+				if req.Method == "GET" &&
+					strings.LastIndex(strings.TrimRight(req.URL.Path, "/"), "/") == 0 && // top-level path
+					!strings.HasPrefix(req.URL.Path, "/_") { // e.g. /_health endpoint
+					return false
+				}
```

Comment:
just to verify, we're okay with these not having some identifying prefix right? ik that came up in discussions though we definitely didn't want it to be a path like `/i/abcdef`